app/views/users/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<% provide :title, t("users.new.title") %>
+<h1>Sign up</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user, url: signup_path) do |f| %>
```

Comment:
anh không set URL được kem

Reply:
form_for trên có nghĩa là tạo ra 1 user_controller và submit đến url signup_path đã được định nghĩa trong router (get "/signup", to: "users#new").
![screenshot from 2018-04-16 11-07-04](https://user-images.githubusercontent.com/38024839/38789736-856ef192-4166-11e8-9141-72d7c41f366b.png)
ở đây signup_path chỉ làm việc với GET user#new.
Nhưng theo e tìm hiểu thì khi ta dùng form_for thì helper trong Rails sẽ tự động dùng POST phù hợp với các cài đặt trong config/router.rb của mình.

Reply:
vậy nên t không cần set url củng được